src/main/java/com/springboot/star_wars_character_popularity/app/controller/VoteController.java

```diff
@@ -62,7 +63,7 @@ public ModelAndView uploadPage(Model model){
     }
 
     @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
-    public ModelAndView saveVote(@ModelAttribute Vote vote, RedirectAttributes redirectAttributes, Model model, @RequestPart(value="image", required = false) MultipartFile multipartFile) throws IOException {
+    public ModelAndView saveVote(@ModelAttribute Vote vote, RedirectAttributes redirectAttributes, Model model, @RequestPart(value="image", required = false) MultipartFile multipartFile) throws IOException, FileNotSavedException {
```

Comment:
If you are creating your own exception, then there is no need for it to be here.